### ISSUE TITLE
Remove explicit dependencies

### ISSUE THREAD

**Issue by realStandal** [LockTech/cerberus#33]
* `babel-loader`
* `eslint-plugin-prettier`
* `null-loader`

---

While cleaning up the project's dependencies (regenerating `yarn.lock` and reinstalling all `node_modules`), it became apparent that [babel-loader](https://github.com/babel/babel-loader) was not being installed through the application's dependencies which require it (see `yarn why` output below). 

This issue is a reminder to remove the explicit dependency on babel-loader from the Web-side of the application. Redwood v0.36 was ***rough*** on Cerberus, so I'm going to wait for the next release to call it an issue on Cerberus' end - or to make noise that it is an issue on Redwood's.

`yarn why babel-loader`

```
❯ yarn why babel-loader
yarn why v1.22.11
[1/4] Why do we have the module "babel-loader"...?
[2/4] Initialising dependency graph...
[3/4] Finding dependency...
[4/4] Calculating file sizes...
=> Found "babel-loader@8.2.2"
info Reasons this module exists
   - "_project_#@redwoodjs#core" depends on it
   - Hoisted from "_project_#@redwoodjs#core#babel-loader"
   - Hoisted from "_project_#@storybook#builder-webpack4#babel-loader"
   - Hoisted from "_project_#@storybook#core-common#babel-loader"
   - Hoisted from "_project_#@redwoodjs#core#babel-timing#babel-loader"
   - Hoisted from "_project_#@storybook#builder-webpack4#@storybook#core-common#babel-loader"
   - Hoisted from "_project_#@redwoodjs#core#@redwoodjs#testing#@storybook#builder-webpack5#babel-loader"
   - Hoisted from "_project_#@redwoodjs#core#@redwoodjs#testing#@storybook#manager-webpack5#babel-loader"
   - Hoisted from "_project_#@redwoodjs#core#@redwoodjs#testing#@storybook#react#@storybook#core#@storybook#core-server#@storybook#builder-webpack4#babel-loader"
   - Hoisted from "_project_#web#@storybook#addon-essentials#@storybook#addon-docs#@storybook#core#@storybook#core-server#@storybook#core-common#babel-loader"
   - Hoisted from "_project_#@redwoodjs#core#@redwoodjs#testing#@storybook#react#@storybook#core#@storybook#core-server#@storybook#manager-webpack4#babel-loader"
   - Hoisted from "_project_#web#@storybook#addon-essentials#@storybook#addon-docs#@storybook#core#@storybook#core-server#@storybook#manager-webpack4#babel-loader"
Done in 1.53s.
```

babel-loader as defined in `yarn.lock` (prior to explicit dependency)

```
babel-loader@8.2.2, babel-loader@^8.1.0, babel-loader@^8.2.2:
  version "8.2.2"
  resolved "https://registry.yarnpkg.com/babel-loader/-/babel-loader-8.2.2.tgz#9363ce84c10c9a40e6c753748e1441b60c8a0b81"
  integrity (redacted? Idk if it's unsafe - probably not yarn.lock is commited but I just wanted to type something)
  dependencies:
    find-cache-dir "^3.3.1"
    loader-utils "^1.4.0"
    make-dir "^3.1.0"
    schema-utils "^2.6.5"
```

**Comment by realStandal** [comment]
`eslint-plugin-prettier` is also gone??

`yarn why eslint-plugin-prettier`:

```
❯ yarn why eslint-plugin-prettier
yarn why v1.22.11
[1/4] Why do we have the module "eslint-plugin-prettier"...?
[2/4] Initialising dependency graph...
[3/4] Finding dependency...
[4/4] Calculating file sizes...
=> Found "eslint-plugin-prettier@3.4.0"
info Reasons this module exists
   - "_project_#@redwoodjs#core#@redwoodjs#eslint-config" depends on it
   - Hoisted from "_project_#@redwoodjs#core#@redwoodjs#eslint-config#eslint-plugin-prettier"
Done in 1.68s.
```

No idea

**Comment by realStandal** [comment]
This issue is resolved after upgrading to RW `0.37.0-rc.60`.
